python/artifacts/README.md

```diff
@@ -0,0 +1,9 @@
+## **/artifacts** folder
+Add your models artifact files needed by your src code at runtime in this directory.
```

Comment:
```suggestion
Add your models artifact files needed by your src code at runtime in this directory. This may include the model's weight files or data that the model needs while running.
```

Reply:
Done